pkg/core/native/crypto_blspoints.go

```diff
@@ -0,0 +1,84 @@
+package native
+
+import (
+	"fmt"
+
+	bls12381 "github.com/consensys/gnark-crypto/ecc/bls12-381"
+	"github.com/nspcc-dev/neo-go/pkg/vm/stackitem"
+)
+
+// Wrappers around bls12381 types that must be used as stackitem.Interop values and
+// implement stackitem.Equatable interface.
+type (
+	G1Affine struct {
+		*bls12381.G1Affine
```

Comment:
I'd either store a `struct` here (not a pointer)...

Reply:
I like the second suggestion, so fixed with https://github.com/nspcc-dev/neo-go/pull/3042#discussion_r1230120207.